static/app/utils/replays/hydrateA11yFrame.tsx

```diff
@@ -1,3 +1,7 @@
+export interface RawA11yResponse {
+  data: RawA11yFrame[];
```

Comment:
there's a `meta` prop in here too it looks like?

Reply:
Its getting popped in another PR.